configuration/frontend/base-config.json

```diff
@@ -1,7 +1,7 @@
 {
   "@openmrs/esm-api": {
     "redirectAuthFailure": {
-      "url": "/pihcore/login.htm"
+      "url": "${openmrsBase}/authenticationui/login/login.page"
```

Comment:
I think this should work, but another approach would be to redirect to something like "/index.htm" and let the authentication module take care of the necessary redirection.  Basically, if you point to any URL that is served by OpenMRS but which is _not_ whitelisted, then you should be redirected to the appropriate login page if you are not yet authenticated.  In the current case, that will be the page you list above, though I suppose it might not always be...